### PR TITLE
Slack connector helm chart

### PR DESCRIPTION
**Pull request by KarolJaksik**
For now, the container image and kyma address have to be set during the installation

### CODE REVIEW

File: chart/slackconnector/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{{- define "slackconnector.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "slackconnector.fullname" -}}
```
Comment (pPrecel) [inline chart/slackconnector/templates/_helpers.tpl]: I think you have to remove define of "slackconnector.fullname" because you never used it

Comment (KarolJaksik) [reply chart/slackconnector/templates/_helpers.tpl]: Done

File: chart/slackconnector/templates/application.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    app: {{ include "slackconnector.name" . }}
+    chart: {{ include "slackconnector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "slackconnector.name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+
+
```
Comment (colunira) [inline chart/slackconnector/templates/application.yaml]: I think you should consider complying with [this](https://helm.sh/docs/chart_best_practices/#whitespace-in-generated-templates) Helm Best Practices suggestion in all your files. 

I'm not also sure if your labels are proper - they seem duplicated, but ask someone if that's right.

Comment (KarolJaksik) [reply chart/slackconnector/templates/application.yaml]: Removed the whitespaces, also not sure about the labels

File: chart/slackconnector/templates/application.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  name: {{ .Release.Name }}-app
+  labels:
+    app: {{ include "slackconnector.name" . }}
+    chart: {{ include "slackconnector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "slackconnector.name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
```
Comment (Szymongib) [inline chart/slackconnector/templates/application.yaml]: Labels `chart`, `release` and `heritage` are basically an older version of `helm.sh/chart`, `app.kubernetes.io/managed-by`, `app.kubernetes.io/instance`. I do not think you need all of them. 
I would suggest sticking to the standard ones https://helm.sh/docs/chart_best_practices/#standard-labels and remove `chart`, `release`, `heritage`.

File: chart/slackconnector/templates/application.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .Release.Name }}-app
```
Comment (Szymongib) [inline chart/slackconnector/templates/application.yaml]: We already know that this is Application based on a Kind. I would suggest using only `{{ .Release.Name }}`

File: chart/slackconnector/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ include "slackconnector.name" . }}
+    chart: {{ include "slackconnector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "slackconnector.name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
```
Comment (Szymongib) [inline chart/slackconnector/templates/deployment.yaml]: Same as in Application, you probably do not need all those labels.

File: chart/slackconnector/templates/service.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ include "slackconnector.name" . }}
+    chart: {{ include "slackconnector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "slackconnector.name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
```
Comment (Szymongib) [inline chart/slackconnector/templates/service.yaml]: Same as in Application, you probably do not need all those labels.

**Pull request merged**